app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -392,3 +396,86 @@ export const mockGetStatus = (status: NimbusExperimentStatus | null) => {
   const { experiment } = mockExperimentQuery("boo", { status });
   return getStatus(experiment);
 };
+
+const fiveDaysAgo = new Date();
+fiveDaysAgo.setDate(fiveDaysAgo.getDate() - 5);
+
+/** Creates a single mock experiment suitable for getAllExperiments queries.  */
+export function mockSingleDirectoryExperimentFactory(
+  overrides: Partial<getAllExperiments_experiments> = {},
+): getAllExperiments_experiments {
+  return {
+    __typename: "NimbusExperimentType",
+    slug: `some-experiment-${Math.round(Math.random() * 100)}`,
+    owner: {
+      __typename: "NimbusExperimentOwner",
+      username: "example@mozilla.com",
+    },
+    monitoringDashboardUrl:
+      "https://grafana.telemetry.mozilla.org/d/XspgvdxZz/experiment-enrollment?orgId=1&var-experiment_id=bug-1668861-pref-measure-set-to-default-adoption-impact-of-chang-release-81-83",
+    name: "Open-architected background installation",
+    status: NimbusExperimentStatus.COMPLETE,
+    featureConfig: {
+      __typename: "NimbusFeatureConfigType",
+      slug: "newtab",
+      name: "New tab",
+    },
+    proposedEnrollment: 7,
+    proposedDuration: 28,
+    startDate: fiveDaysAgo.toISOString(),
+    endDate: new Date(Date.now() + 12096e5).toISOString(),
```

Comment:
Q: What's the `12096e5`? 14 days?
Wondering if we should convert it to some const so it's more self-describing.

Reply:
That was me 🙈 You are correct, it's "2 weeks from now". I'm fine if we want to make this more explicit.

Reply:
i could use my new `addDays` helper now!!

Reply:
Mind if i leave this for now / we could refactor both at once at some point if we do improvements on mocks?

Reply:
I'm fine with ignoring this for now. I vote that we just #yolomerge this in and we can file TODO issues for when we inevitably tweak the UI.